### ISSUE TITLE
Is there a way to get the token usage after a completion request?

### ISSUE THREAD

**Issue by Thimows** [arshad-yaseen/monacopilot#70]
Hi all,

Is there a way after the AI completion request in our backend to access the used token amount from openAI (or Antrophic if that is used instead)? That would be great as we need to save the token usage for every request to the user for billing purposes.

Thanks!

**Comment by arshad-yaseen** [comment]
Hey, There is no way to get it yet. But, adding it very soon. Wait for it!

**Comment by arshad-yaseen** [comment]
Released `v0.13.1` 🚀

Please update Monacopilot to this version. 

Here's how to do it:

```javascript
const { completion, error, raw } = await copilot.complete({
  body: req.body,
});

// Process the raw LLM response
if (raw) {
  calculateCost(raw.usage.total_tokens);
}
```

> **Note**: The `raw` can be undefined if an error occurred, which happens when `error` is present

For more context, refer to the [API Handler code example](https://github.com/arshad-yaseen/monacopilot?tab=readme-ov-file#api-handler).

**Comment by Thimows** [comment]
Thanks for the super fast update @arshad-yaseen!!
